### PR TITLE
Updated "Microsoft Edge Privacy Whitepaper"

### PR DESCRIPTION
**Pull request by vchapel**
Acrolinx issue fixes.

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -3,187 +3,219 @@ description: Microsoft Edge Privacy Whitepaper
...
+To help us improve Microsoft products and services, anonymous and aggregated diagnostic data is stored for up to two years. Because the diagnostic data is not collected from or stored with your Microsoft account, the diagnostic data may not be viewed or deleted from your [Microsoft privacy dashboard][MicrosoftAccountPrivacy]. To delete the diagnostic data on Windows 10 and later, navigate to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback** and select **Delete** under **Delete diagnostic data**. The delete diagnostic data functionality is only supported on Windows 10 version 1803 or later. For more information, navigate to [Diagnostics, feedback, and privacy in Windows 10 and later][MicrosoftSupport4468236].
```
Comment (dalenaMSFT) [inline microsoft-edge/privacy-whitepaper/index.md:149]: we should not be using the term "anonymous".  We can change de-identified to "we remove personal identifiers..." or some variant like that.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -371,35 +371,35 @@ Microsoft Edge asks you if you want to store your payment info if payment autofi
...
- 
-The personalization feature is only available for users with a non-child Microsoft account. The feature is not available for users signed in to Microsoft Edge with a work or school account.  
+If you allow personalization, the Microsoft Edge team collects and uses your Microsoft Edge browsing history to personalize experiences and advertising on [Bing][BingMain], Microsoft News, and other Microsoft services.  Personalization provides more relevant and useful search results, ads, and news content.  For example, if the Microsoft Edge team determines based on your browsing that you prefer a particular store, the ads you see may be for that store.  Similarly, if you frequently look at travel blogs and read travel articles, your news feed may include news content about traveling.
 
-Your browsing history is collected and used for personalization only if all four conditions are met.  
+The personalization feature is only available for users with a non-child Microsoft account. The feature is not available for users signed in to Microsoft Edge with a work or school account.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper/index.md:384]: Did Dalen approve this change?

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -371,35 +371,35 @@ Microsoft Edge asks you if you want to store your payment info if payment autofi
...
 
-* You are signed in to a non-child Microsoft account. 
+Your browsing history is collected and used for personalization only if all four conditions are met.
```
Comment (Reezaali) [inline microsoft-edge/privacy-whitepaper/index.md:386]: Approval received?

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:384]: Yes

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (vchapel) [reply microsoft-edge/privacy-whitepaper/index.md:386]: Yes

**Pull request merged**